browser/js/actions/AuthActions.js

```diff
@@ -35,50 +44,29 @@ export function signup (credentials) {
   }
 }
 
-export function loginSuccess (user) {
-  return {
-    type: AUTH_LOGIN_SUCCESS,
-    id: user.id,
-    user
-  }
-}
-
 export function getLoggedInUser () {
   return (dispatch, getState)  => {
     let user = getState().session.user
     if (user) {
-      dispatch(receivedLoggedInUser(user))
+      dispatch(userReceived(user))
```

Comment:
Not sure I'm personally crazy about this event-based architecture I'm seeing, but you all seem to be executing it pretty well and establishing solid patterns.